content/docs/cml/cml-with-npm.md

```diff
@@ -0,0 +1,40 @@
+# CML with NPM
+
+In the above examples, CML is pre-installed in a custom Docker image, which is
+pulled by a CI runner. You can also install CML as a package:
+
+```bash
+npm i -g @dvcorg/cml
+```
+
+You may need to install additional dependencies to use DVC plots and Vega-Lite
+CLI commands:
+
+```bash
+sudo apt-get install -y libcairo2-dev libpango1.0-dev libjpeg-dev libgif-dev \
+          librsvg2-dev libfontconfig-dev
```

Comment:
```suggestion
sudo apt-get install -y libcairo2-dev libpango1.0-dev libjpeg-dev \
          libgif-dev librsvg2-dev libfontconfig-dev
```

Reply:
BTW these are to avoid horizontal scrolling:

![image](https://user-images.githubusercontent.com/1477535/102681556-dda68400-4187-11eb-9900-f83e2bbe9b00.png)